app/js/lib/speech-controller.js

```diff
@@ -102,14 +98,6 @@ export default class SpeechController extends EventDispatcher {
     this[p.speechSynthesis].speak(text);
   }
 
-  [p.initialiseSpeechRecognition]() {
-    return fetch(this[p.wakewordModelUrl])
-      .then((response) => response.json())
-      .then((model) => {
-        this[p.wakewordRecogniser].loadModel(model);
-      });
-  }
-
   [p.startListeningForWakeword]() {
     this.emit(EVENT_INTERFACE[0], { type: EVENT_INTERFACE[0] });
     this[p.idle] = true;
```

Comment:
I don't think we should go back to that library yet, the more I learn about speech recognition the more inadequate it becomes for the way we want it to work.  If we really want it back, it's in the repositories history.